src/main/webapp/WEB-INF/views/car/drivers_of_car.jsp

```diff
@@ -0,0 +1,34 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>All drivers</title>
+</head>
+<body>
+<h1>All drivers</h1>
```

Comment:
what is the difference between this JSP and `car/drivers_of_a_car`?

Reply:
there are different title and the second one returns user to all cars list while first returns to index.jsp